test/integration/package_int_test.go

```diff
@@ -594,6 +594,9 @@ func (suite *PackageIntegrationTestSuite) TestUpdate() {
 }
 
 func (suite *PackageIntegrationTestSuite) TestRuby() {
+	// TODO: Re-enable this test when Ruby is supported on the Platform
+	// https://activestatef.atlassian.net/browse/DX-2384
+	suite.T().Skip("Ruby is currently not building on the Platform")
```

Comment:
Nit: technically it works on Linux. It's only Windows that's causing a failure (since macOS is skipped).

Reply:
Good point.